hw/ip/otbn/util/yaml_to_doc.py

```diff
@@ -263,24 +264,31 @@ def render_insn(insn: Insn, impl: Optional[str], heading_level: int) -> str:
     # Optional documentation (using existing markdown formatting). Add a blank
     # line afterwards to separate from the syntax and operand table.
     if insn.doc is not None:
-        parts.append(insn.doc + '\n\n')
-
-    # Syntax example: either given explicitly or figured out from operands
-    parts.append("```\n")
-    parts.append(insn.mnemonic.upper() + ('' if insn.glued_ops else ' '))
-    parts.append(insn.syntax.render_doc())
-    parts.append("\n```\n\n")
+        parts.append(insn.doc + '\n')
+    parts.append('\n')
 
     # If this came from the RV32I instruction set, say so.
     if insn.rv32i:
         parts.append('This instruction is defined in the RV32I instruction set.\n\n')
 
-    # Show any trailing documentation (stuff that should come after the syntax
-    # example but before the operand table).
-    if insn.trailing_doc is not None:
+    # A list of errors that the instruction might cause.
+    if insn.errs is not None:
+        parts.append(subhead + 'Errors\n')
+        if not insn.errs:
+            parts.append('{} cannot cause any synchronous errors.\n'.format(mnem))
+        else:
+            parts.append('{} might cause the following synchronous errors:\n'
+                         .format(mnem))
```

Comment:
Two comments regarding the wording here:

- We don't use or define the term "synchronous errors" anywhere in the spec.
- The errors listed here are only the ones that are instruction-related/software-related. There might be other errors which are happening when accessing resources during the execution of the instruction, e.g. an integrity error when reading the register file or the data memory.

I left a note at https://github.com/lowRISC/opentitan/pull/8243#issuecomment-921851464 to discuss the terminology, but maybe the following change will do the trick:

```suggestion
            parts.append('{} might cause the following software errors:\n'
                         .format(mnem))
```

Reply:
Sounds like a good idea for now, at least.